src/Modal.js

```diff
@@ -166,59 +198,70 @@ class Modal extends React.Component {
 
   renderChildren() {
     const {
-      className,
       wrapClassName,
       modalClassName,
       backdropClassName,
-      contentClassName,
       cssModule,
       isOpen,
-      size,
-      backdrop,
-      children,
-      ...attributes
-    } = omit(this.props, ['toggle', 'keyboard', 'onEnter', 'onExit', 'zIndex']);
+      backdrop
+    } = this.props;
+
+    const modalAttributes = {
+      onClickCapture: this.handleBackdropClick,
+      onKeyUp: this.handleEscape,
+      style: { display: 'block' },
+      tabIndex: '-1'
+    };
+
+    if (this.hasTransition()) {
+      const modalTransitionTimeout = 300;
+      const backdropTransitionTimeout = 150;
```

Comment:
I thought that was a good idea. I added that in this commit: https://github.com/reactstrap/reactstrap/pull/339/commits/a2fb9154d7da44b9540d888472193a591da7ed6d

Let me know what you think. Thank you for taking a look!